plugins/pulp_docker/plugins/migrations/0003_tag_schema_change.py

```diff
@@ -0,0 +1,20 @@
+from pulp.server.db.connection import get_collection
+
+
+def migrate(*args, **kwargs):
+    """
+    Add schema_version to the tag collection.
+    """
+
+    schema_version_key = 'schema_version'
+    collection = get_collection('units_docker_tag')
+    # drop old index due to unit_keys fields change
+    index_info = collection.index_information()
+    old_index = 'name_1_repo_id_1'
```

Comment:
in my understanding mongo will automatically create the new index, at least it happened in my case without doing anything